Build/Scripts/nunit2junit.ps1

```diff
@@ -0,0 +1,36 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [System.IO.DirectoryInfo]
+    $InputDirectory,
+
+    [Parameter(Mandatory = $true)]
+    [System.IO.DirectoryInfo]
+    $OutputDirectory,
+    
+    [System.IO.FileInfo]
+    $XSL = $((Resolve-Path (Join-Path $PSScriptRoot 'nunit2junit.xslt')).Path)
+)
+
+$ErrorActionPreference = "Stop"
+
+Write-Host "Using transformer '$($XSL.FullName)'"
+Write-Host "Converting NUnit XML files from folder '$($InputDirectory.FullName)'"
+Write-Host "Outputting JUnit XML files into folder '$($OutputDirectory.FullName)'"
+Write-Host ""
+
+$settings = New-Object System.Xml.Xsl.XsltSettings
+$resolver = New-Object System.Xml.XmlUrlResolver
+$xslt = New-Object System.Xml.Xsl.XslCompiledTransform
+
+$settings.EnableScript = 1
+$xslt.Load($XSL.FullName, $settings, $resolver)
+
+Get-ChildItem $InputDirectory -Filter *.xml |
+Foreach-Object {
+    Write-Host "Working on '$($_.Directory)' :: '$($_.Name)'"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File 'nunit2junit.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5097765)